cumulusci/cli/config.py

```diff
@@ -50,6 +50,24 @@ def _load_keychain(self):
             self.keychain = self.keychain_class(self.project_config, self.keychain_key)
             self.project_config.set_keychain(self.keychain)
 
+    def alert(self, message=None):
+        try:
+            if self.project_config.dev_config__no_alert:
+                return
+        except AttributeError:
+            pass
```

Comment:
What's the case where there's an AttributeError here? Is there test coverage for that case? Can we avoid it?

Reply:
if dev_config or no_alert are not yet defined. the test where the alert "succeeds" goes down the except & pass.

Reply:
The except clause is showing as uncovered in coveralls. I think because BaseConfig's `__getattr__` returns None when a key isn't found rather than raising AttributeError. I think we just need to test the case where project_config is not set yet.

The next condition (`if message is None: message = "We need your attention!"`) is also uncovered. I would just make `message` a required arg for the function.